bundles/core/org.eclipse.smarthome.core.test/META-INF/MANIFEST.MF

```diff
@@ -18,4 +18,6 @@ Import-Package: groovy.lang,
  org.junit,
  org.junit.runner,
  org.junit.runners,
- org.osgi.service.cm
+ org.osgi.service.cm,
+ org.mockito
+Require-Bundle: org.junit,org.mockito,org.hamcrest
```

Comment:
My explanation above was wrong in one point: I did introduce the Required-Bundle to fully support hamcrest (not mockito). I tried with one of the other affected bundles (org.eclipse.smarthome.io.rest.core.test) and there the change will lead to compile errors since hamcrest is not fully available. Here only org.hamcrest.CoreMatchers is used so no Required-Bundle is required.